src/validation/schema.ts

```diff
@@ -16,15 +16,28 @@ import { isObjectValue } from './util'
 export interface ValidationOptions {
```

Comment:
note: Now looking back I'd have called this `v0Support` instead of `ValidationOptions`. This is where we'd also pass any config to still support deprecated features. 

Another one that comes to my mind is [dropping `type` attr in favor of of `inputType`](https://linear.app/remote/issue/DEVXP-2862/deprecate-type-attr-on-field). If we have done that already, then it would look like this:

```js
const form = createHeadlessForm(schema, {
  v0Support: {
      includeTypeAttr: true,
  }
}
```

Reply:
indeed 👍

Reply:
https://github.com/remoteoss/json-schema-form/pull/212/files